frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -298,6 +293,34 @@ dbg:guard(FileManagerMenu, 'setUpdateItemTable',
         end
     end)
 
+function FileManagerMenu:exitOrRestart(callback)
+    if SetDefaults.settings_changed then
+        UIManager:show(ConfirmBox:new{
+            text = _("You have unsaved default settings. Save them now?\nTap \"Cancel\" to return to KOReader."),
+            ok_text = _("Yes"),
```

Comment:
I am OK to "revert" the OK / Cancel buttons manually, but should not we make changes directly in ConfirmBox?